.github/workflows/linux.yml

```diff
@@ -33,8 +33,9 @@ jobs:
       run: sudo apt-get install -y g++-4.8
     - name: Start Xvfb
       run: |
-        Xvfb :99 &
-        disown -ar
-        echo "::set-env name=DISPLAY:::99"
+        sudo apt-get install libwoff1 libopus0 libwebp6 libwebpdemux2 libenchant1c2a libgudev-1.0-0 libsecret-1-0 libhyphen0 libgdk-pixbuf2.0-0
```

Comment:
Taken from https://github.com/microsoft/playwright/actions/runs/25967532/workflow

Reply:
These deps shouldn't be needed for our tests.